src/charm.py

```diff
@@ -56,6 +61,26 @@ def _manage_prometheus_bind_exporter_service(self):
         ])
         logger.info("prometheus-bind-exporter has been reconfigured")
 
+    def _render_grafana_dashboard(self) -> str:
```

Comment:
Since charmhelpers is not used (or discourage) in the Operator Framework, a bug should be filed to create reusable helpers. For example. this method could call a generic helper function (maybe in the lib folder?), passing the context, template_dir and template_name via args.